sentry_sdk/integrations/redis.py

```diff
@@ -15,6 +15,43 @@
 _MULTI_KEY_COMMANDS = frozenset(["del", "touch", "unlink"])
 
 
+def patch_redis_pipeline(pipeline_cls, parse_command_fn):
+    # type: (Any) -> None
+    old_execute = pipeline_cls.execute
+
+    def sentry_patched_execute(self, *args, **kwargs):
+        # type: (Any, *Any, **Any) -> Any
+        hub = Hub.current
+
+        if hub.get_integration(RedisIntegration) is None:
+            return old_execute(self, *args, **kwargs)
+
+        with hub.start_span(op="redis", description="redis.pipeline.execute") as span:
+            with capture_internal_exceptions():
+                print(self.command_stack)
+                commands = [parse_command_fn(c) for c in self.command_stack[:10]]
+                if len(self.command_stack) > 10:
+                    commands.append("...")
+                span.set_data(
+                    "commands",
+                    {"count": len(self.command_stack), "first_ten": commands},
+                )
+
+            return old_execute(self, *args, **kwargs)
+
+    pipeline_cls.execute = sentry_patched_execute
+
+
+def _parse_redis_command(command):
+    # type: (Any) -> str
+    return " ".join(map(str, command[0]))
```

Comment:
i would align this with the code at the bottom of this file. there we use `repr` instead of `str` and no list comprehension/slicing. Perhaps you find a way to factor this out.

Reply:
Done. I factored out the constant.